tests/cypress/e2e/registration/registrationModalRedirect.cy.ts

```diff
@@ -0,0 +1,24 @@
+/// <reference types="cypress" />
+
+describe('Registration modal redirect flow', { testIsolation: false }, () => {
+  const devices = ['desktop', 'mobile']
+
+  devices
+    .filter((device) => Cypress.env('devices')[`${device}`])
+    .forEach((device) => {
+      context(device, Cypress.env('resolution')[`${device}`], () => {
+
+        it('Checking if registration modal window is open.', () => {
+          cy.visit('/mestske-sluzby/stanovisko-k-investicnemu-zameru')
+          cy.get(`#${device}-navbar`).invoke('attr', 'style', 'display: none')
```

Comment:
```suggestion
          cy.hideNavbar(device)
```
Please create a custom command (cy.hideNavbar()) that will hide the navbar. The input will be device. Then please replace this command wherever it is used. Use beforeEach() inside this file where you will use it. Same in the following test which is included in this PR.